test/failure/bad-hcom-step.prl

```diff
@@ -1,10 +0,0 @@
-Thm BoolSym : [
-  (-> 
-   [a b : bool]
-   (path {_} bool a b)
-   (path {_} bool b a))
-] by [
-  { lam a. lam b. lam p. <i> 
-     `(hcom{0 ~> 1} bool ,a [i=0 {j} (@ ,p j)] [i=1 {_} ,a])};
-  [auto-step; head-expand]
```

Comment:
This is probably because I forgot to change bool to wool... Glad that it's useful!